worldwind/src/commonMain/kotlin/earth/worldwind/shape/AbstractShape.kt

```diff
@@ -44,14 +45,36 @@ abstract class AbstractShape(override var attributes: ShapeAttributes): Abstract
     override var highlightAttributes: ShapeAttributes? = null
     override var isHighlighted = false
     var maximumIntermediatePoints = 10
-    protected lateinit var activeAttributes: ShapeAttributes
-    protected var isSurfaceShape = false
+    lateinit var activeAttributes: ShapeAttributes
+        protected set
+    var isSurfaceShape = false
```

Comment:
Smae here. Do we still need to hava it public?

Reply:
Yes, still used in PathSet to get attributes